nats/main.go

```diff
@@ -52,7 +52,7 @@ See 'nats cheat' for a quick cheatsheet of commands`
 	ncli.Flag("user", "Username or Token").Envar("NATS_USER").PlaceHolder("USER").StringVar(&opts.Username)
 	ncli.Flag("password", "Password").Envar("NATS_PASSWORD").PlaceHolder("PASSWORD").StringVar(&opts.Password)
 	ncli.Flag("token", "Token").Envar("NATS_TOKEN").PlaceHolder("TOKEN").StringVar(&opts.Token)
-	ncli.Flag("connection-name", "Nickname to use for the underlying NATS Connection").Default("NATS CLI Version " + getVersion()).PlaceHolder("NAME").StringVar(&opts.ConnectionName)
+	ncli.Flag("connection-name", "Nickname to use for the underlying NATS Connection").Default("NATS CLI Version " + getVersion()).StringVar(&opts.ConnectionName)
```

Comment:
I think we should revert this one it can be a bit distracting 

```
      --connection-name="NATS CLI Version v0.2.3-0.20250704133146-7dae391fe8c7+dirty"
                                Nickname to use for the underlying NATS Connection
```

Even on release this would be quite a long default to show